packages/build-info/src/package-managers/detect-package-manager.ts

```diff
@@ -23,22 +25,25 @@ export type PkgManagerFields = {
 }
 
 /** The definition of all available package managers */
-const AVAILABLE_PACKAGE_MANAGERS: Record<PkgManager, PkgManagerFields> = {
+export const AVAILABLE_PACKAGE_MANAGERS: Record<PkgManager, PkgManagerFields> = {
   [PkgManager.YARN]: {
     name: PkgManager.YARN,
     installCommand: 'yarn install',
+    runCommand: 'yarn',
```

Comment:
I think this potentially could be exploited if the script-name is set to `install`, `uninstall x`, etc. I would use this just to be safe:
```suggestion
    runCommand: 'yarn run',
```

https://classic.yarnpkg.com/lang/en/docs/cli/run/

Reply:
Incorporated in https://github.com/netlify/build/pull/4876/commits/6561b05307bde3490e54d00197aa8f75a97a04c9